Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.9.6-slim-buster as base 
+
+RUN apt-get update \ 
+    && apt-get -y install make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
```

Comment:
I don't think you need all this, just curl.

The base image has the other dependencies of Python installed already.